release-notes.md

```diff
@@ -1,5 +1,17 @@
 # Kajam-11.9.0 #
 
+## add roles field on widget and set permissions accordingly ##
+ - Issue Type: Bug
+ - Issue ID: PLAT-3728
+### Installation ###
+None.
+### Configuration ###
+None.
+#### Known Issues & Limitations ####
+None.
```

Comment:
known limitation should be that we currently du not support passing multiple roles on the Widget object